rskj-core/src/main/java/co/rsk/net/SyncProcessor.java

```diff
@@ -62,12 +66,14 @@ public class SyncProcessor implements SyncEventsHandler {
     private final EthereumListener ethereumListener;
 
     private final PeersInformation peersInformation;
-    private final Map<Long, MessageType> pendingMessages;
+    private final Map<Long, MessageInfo> pendingMessages;
     private final AtomicBoolean isSyncing = new AtomicBoolean();
 
     private volatile long initialBlockNumber;
     private volatile long highestBlockNumber;
 
+    private volatile long lastDelayWarn = System.currentTimeMillis();
```

Comment:
```suggestion
    private volatile long lastDelayWarn = 0;
```
SyncProcessor instance creation doesn't mean that the sync process has started.

Reply:
Yes, but this is irrelevant for the logic, no? The first time a delay is detected it will be printed anyway (with either values), and then not again until the time passes.